cmake/FindCMSIS.cmake

```diff
@@ -313,6 +313,13 @@ foreach(COMP ${CMSIS_FIND_COMPONENTS_FAMILIES})
         set(CMSIS_${RTOS_COMP}_FOUND TRUE)
     endforeach()
 
+    if(NOT (TARGET CMSIS::STM32::${FAMILY}${CORE_C}::DSP}))
```

Comment:
At this point, do you know for sure that we do have the DSP for all families ? otherwise we might want to check first

Reply:
Hi @Hish15, 
Sorry for taking so long to respond on this one. I went through and validated that every family has the ability to pull in DSP. Would you like me to write a unit test to make sure this remains the case?